src/common/legacy_config_opts.h

```diff
@@ -383,6 +383,7 @@ OPTION(crush_location_hook_timeout, OPT_INT)
 
 OPTION(objecter_tick_interval, OPT_DOUBLE)
 OPTION(objecter_timeout, OPT_DOUBLE)    // before we ask for a map
+OPTION(objecter_homeless_timeout, OPT_FLOAT) // timeout homeless sessions
```

Comment:
Would TYPE_SECS work better than TYPE_FLOAT?

(It doesn't support units smaller than 1second, though, it looks like.)